financial-connections/src/main/java/com/stripe/android/financialconnections/features/linkstepupverification/LinkStepUpVerificationScreen.kt

```diff
@@ -111,16 +100,23 @@ private fun LinkStepUpVerificationLoaded(
     onClickableTextClick: (String) -> Unit
 ) {
     val focusManager = LocalFocusManager.current
-    val focusRequester: FocusRequester = remember { FocusRequester() }
-    LaunchedEffect(Unit) { focusRequester.requestFocus() }
     val textInputService = LocalTextInputService.current
+
+    val focusRequester: FocusRequester = remember { FocusRequester() }
+    var shouldRequestFocus by rememberSaveable { mutableStateOf(false) }
+
+    if (shouldRequestFocus) {
+        LaunchedEffect(Unit) { focusRequester.requestFocus() }
+    }
```

Comment:
This didn’t cause issues before, but is now failing tests with `FocusRequester not initialized` errors. The workaround is to defer the request until the text field is displayed.

I can look into creating a `SafeFocusRequester` or similar if you want.